lib/travis/logs/receive/queue.rb

```diff
@@ -27,64 +27,64 @@ def subscribe
 
         private
 
-          def consumer
-            Travis::Amqp::Consumer.jobs(name, channel: { prefetch: prefetch })
-          end
+        def consumer
+          Travis::Amqp::Consumer.jobs(name, channel: { prefetch: prefetch })
+        end
 
-          def prefetch
-            Travis::Logs.config.amqp.prefetch
-          end
+        def prefetch
+          Travis::Logs.config.amqp.prefetch
+        end
 
-          def receive(message, payload)
-            smart_retry do
-              if payload = decode(payload)
-                Travis.uuid = payload.delete('uuid')
-                handler.run(payload)
-              end
+        def receive(message, payload)
+          smart_retry do
+            if payload = decode(payload)
+              Travis.uuid = payload.delete('uuid')
+              handler.run(payload)
             end
-            message.ack
-          rescue => e
-            log_exception(e, payload)
-            message.reject(requeue: true)
-            Metriks.meter("#{METRIKS_PREFIX}.receive.retry").mark
-            error "[queue:receive] message requeued"
           end
+          message.ack
+        rescue => e
+          log_exception(e, payload)
+          message.reject(requeue: true)
```

Comment:
Not sure, but Bunny doesn't not have `#reject` on message. Some time ago, I was created https://github.com/travis-ci/travis-support/pull/15 and there is extension for `reject`: https://github.com/final-ci/travis-support/commit/61d556e38746241e2ad22f3521de1454b557d4b9 - you wish I can prepare pull request with this commit.